src/js/components/service-requests/submit-service-request.js

```diff
@@ -81,19 +81,15 @@ export class SubmitServiceRequest {
                 )
                 .done(() => {
                   this.renderSubmitted();
+                })
+                .fail((a, b) => {
+                  this.renderFailure($submitButton);
+                  console.error(a, b);
                 });
             }
           })
           .fail((a, b) => {
-            this.$element.empty().append(
-              new StatusMessage({
-                text: "Error while submitting request.",
-                status: "failure",
-              }).render()
-            );
-            $submitButton
-              .attr({ value: "Submit", disabled: false })
-              .removeClass("button--disabled");
+            this.renderFailure($submitButton);
```

Comment:
It feels a bit like this is doing double duty 😄  Could we do:

```suggestion
            this.renderFailure();
            $submitButton
               .attr({ value: "Submit", disabled: false })
              .removeClass("button--disabled");
```